pyre/check.py

```
@@ -1774,6 +1775,200 @@ def default_binary(backend):
+    failed to resolve from the repository root and the whole mechanism
+    contributed nothing but a dependence on which profiles and targets happened
+    to have been built.
+
+    One class is knowingly outside the set: an input a build script reads from
+    outside its crate. The only one in the tree is the `lib-python/3` closure
+    `pyre-interpreter/build.rs` embeds, and it is guarded by `wasm_vfs`, a
+    feature no artefact this script measures is built with. Enabling it for the
+    wasm-host build would mean adding that closure here.
+    """
+    try:
+        listing = subprocess.run(
```

Ruff (0.16.1)

[warning] `subprocess.run` without explicit `check` argument

Add explicit `check=False`

(PLW1510)